dejure.org.js

```diff
@@ -0,0 +1,244 @@
+{
+	"translatorID": "16c7d938-5f77-4fb5-99a1-bcec6fdafe84",
+	"label": "dejure.org",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://dejure\\.org/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-01-14 14:59:56"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/dienste/vernetzung/')>-1) {
+		if (getSearchResults(doc, true)) {
+			return "multiple";
+		} else {
+			return "case";
+		}
+	} else if (url.indexOf('/gesetze/')>-1 && url.indexOf('.html')>-1) {
+		return "statute";
+	}
+}
+
+
+//We only handle the cases that there are multiple cases on the
+//same pages in multiples.
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var itemsDoc = [];
+	var found = 0;
+	var rows = ZU.xpath(doc, '//div[contains(@class, "rspr_inhalt")]');
+	for (var i=0; i<rows.length; i++) {
+		var title = ZU.xpathText(rows[i], './/td[contains(@class, "urteilszeile")]');
+		if (!title) continue;
+		found++;
+		items[i] = title;
+		itemsDoc[i] = rows[i];
+	}
+	if (checkOnly) {
+		if (found>1) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	return found ? [items, itemsDoc] : false;
+}
+
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if (type == "multiple") {
+		var results = getSearchResults(doc, false);
+		Zotero.selectItems(results[0], function (items) {
+			if (!items) {
+				return true;
+			}
+			for (var i in items) {
+				scrape(results[1][i], url, "case");
```

Comment:
Yes, that is right. See for example http://web.archive.org/web/20160324073816/http://dejure.org/dienste/vernetzung/rechtsprechung?Text=NJW%202003,%203406 (Are you sure you don't have access? I thought it would be open for everyone...)